chapter-1/exercises/1.10.js

```diff
@@ -0,0 +1,132 @@
+//Exercise 1.10. The following function computes a mathematical function called Ackermann’s function.
+
+function A(x,y) {

```

Comment:
Same notes on convention like https://github.com/tabdulradi/sicp/pull/3/files
I know you copied this code. But this teaches you not to paste any code blindly.